_layouts/project.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+---
+<article class="post">
+  <header class="post-header">
+    <h1 class="project-title">{{ page.title | escape}} </h1>
+    <h2 class="project-creator-names">{{ page.creator_names | escape}}</h2>
```

Comment:
I've given this my best shot. I may need to manipulate the _config.yml file to get it to appear as a horizontal list instead of a vertical list. (As explained here: http://jch.penibelst.de/) Newlines seem to be built into loops.